packaging/linux/openra-join-servers.desktop

```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
+Type=Application
+Version=1.0
+Name=OpenRA
+GenericName=Real Time Strategy Game
+Comment=Reimagining of early Westwood Games
+Icon=openra
+Exec=openra Launch.Connect=%U
+Terminal=false
+StartupNotify=false
```

Comment:
As discussed in the replay handling pr, this will probably cause problems on some distros.